data/import/vocabularies/schema.txt

```diff
@@ -0,0 +1,2805 @@
+ Clip|Category
+ ClothingStore|Category
+ CoOp|Category
+ Code|Category
+ CohortStudy|Category
+ Collection|Category
+ CollectionPage|Category
+ CollegeOrUniversity|Category
+ ComedyClub|Category
+ ComedyEvent|Category
+ ComicCoverArt|Category
+ ComicIssue|Category
+ ComicSeries|Category
+ ComicStory|Category
+ Comment|Category
+ CommentAction|Category
+ CommentPermission|Category
+ CommunicateAction|Category
+ CommunityHealth|Category
+ CompilationAlbum|Category
+ CompleteDataFeed|Category
+ Completed|Category
+ CompletedActionStatus|Category
+ CompoundPriceSpecification|Category
+ ComputerLanguage|Category
+ ComputerStore|Category
+ ConfirmAction|Category
+ Consortium|Category
+ ConsumeAction|Category
+ ContactPage|Category
+ ContactPoint|Category
+ ContactPointOption|Category
+ ContagiousnessHealthAspect|Category
+ Continent|Category
+ ControlAction|Category
+ ConvenienceStore|Category
+ Conversation|Category
+ CookAction|Category
+ Corporation|Category
+ CorrectionComment|Category
+ Country|Category
+ Course|Category
+ CourseInstance|Category
+ Courthouse|Category
+ CoverArt|Category
+ CovidTestingFacility|Category
+ CreateAction|Category
+ CreativeWork|Category
+ CreativeWorkSeason|Category
+ CreativeWorkSeries|Category
+ CreditCard|Category
+ Crematorium|Category
+ CriticReview|Category
+ CrossSectional|Category
+ CssSelectorType|Category
+ CurrencyConversionService|Category
+ DDxElement|Category
+ DJMixAlbum|Category
+ DVDFormat|Category
+ DamagedCondition|Category
+ DanceEvent|Category
+ DanceGroup|Category
+ DangerousGoodConsideration|Category
+ DataCatalog|Category
+ DataDownload|Category
+ DataFeed|Category
+ DataFeedItem|Category
+ DataType|Category
+ Dataset|Category
+ Date|Category
+ DateTime|Category
+ DatedMoneySpecification|Category
+ DayOfWeek|Category
+ DaySpa|Category
+ DeactivateAction|Category
+ DecontextualizedContent|Category
+ DefenceEstablishment|Category
+ DefinedRegion|Category
+ DefinedTerm|Category
+ DefinedTermSet|Category
+ DefinitiveLegalValue|Category
+ DeleteAction|Category
+ DeliveryChargeSpecification|Category
+ DeliveryEvent|Category
+ DeliveryMethod|Category
+ DeliveryTimeSettings|Category
+ Demand|Category
+ DemoAlbum|Category
+ DemoGameAvailability|Category
+ Dentist|Category
+ Dentistry|Category
+ DepartAction|Category
+ DepartmentStore|Category
+ DepositAccount|Category
+ Dermatologic|Category
+ Dermatology|Category
+ DesktopWebPlatform|Category
+ DiabeticDiet|Category
+ Diagnostic|Category
+ DiagnosticLab|Category
+ DiagnosticProcedure|Category
+ Diet|Category
+ DietNutrition|Category
+ DietarySupplement|Category
+ DigitalAudioTapeFormat|Category
+ DigitalDocument|Category
+ DigitalDocumentPermission|Category
+ DigitalDocumentPermissionType|Category
+ DigitalFormat|Category
+ DigitalPlatformEnumeration|Category
+ DisabilitySupport|Category
+ DisagreeAction|Category
+ Discontinued|Category
+ DiscoverAction|Category
+ DiscussionForumPosting|Category
+ DislikeAction|Category
+ Distance|Category
```

Comment:
There also is a lowecase `distance` earlier on in the text file, of type Text. Is that correct?

Reply:
I guess so. [schema:Distance](https://schema.org/Distance) would be for identifying categories and [schemadistance](https://schema.org/distance) for property usage. I downloaded version 14 as CSV from schema.org